glossary.yml

```diff
@@ -2772,6 +2776,13 @@
     term: ክፍል
     def: >
       በዕቃ ተኮር ፕሮግራም (#oop) ውስጥ, መረጃዎችን እና ክንውኖች ([ዘዴዎች](#method)። ፕሮግራሙ ከዚያ ምክኒያት የሚጠቀምበት [ኮንሰርተር](#constructor) ከነዚህ ጋር [አካል](#object) ለመፍጠር ባህሪያት እና ዘዴዎች. ፕሮግራም አዘጋጆች በአብዛኛው ጀነሬክ ወይም እንደገና ሊጠቀሙ የሚችሉ ናቸው ባህሪ በ[ወላጅ መደብ](#parent_class)፣ እና በዝርዝር ወይም በግልጽ ጠባይ በ[ልጆች መደብ](#child_class).
+  uk:  
+    term: клас
+    def: >
+      У [об'єктно-орієнтованому програмуванні](#oop) класи є шаблонами, за якими створюються об’єкти. 
+      Вони допомагають групувати разом пов’язані змінні та функції (названі [методами](#method)). 
+      Програмісти зазвичай розміщують загальну або багаторазову поведінку в [базових класах](#parent_class), 
```

Comment:
Could you please try to formulate better "розміщують загальну або багаторазову поведінку"? I would suggest to replace поведінку -> функціональність, and also find different words for розміщують and багаторазову (it may be non-obvious what does "багаторазову поведінку" mean at all)